lib/auth/github.go

```diff
@@ -539,17 +682,12 @@ func (a *Server) createGithubUser(ctx context.Context, p *createUserParams, dryR
 
 // populateGithubClaims retrieves information about user and its team
 // memberships by calling Github API using the access token
```

Comment:
technically it no longer retrieves information about team memberships by calling the GitHub API since we're now passing the teams in.

It's a little bit awkward that we're passing in a client that fetches some of the info we need, and then passing in the teams instead of just using the client to get them. Can we just pass in the username as well and then there's no need for a client here? (It would be cleaner and easier to test)

Reply:
I agree that was a bit awkward, done